src/api/content-strings/XLSX.ts

```diff
@@ -170,6 +178,8 @@ export const XLSX = {
     [WEBSITE]: 'Exporter Company website (optional)',
     [WILL_EXPORT_WITH_CODE_OF_CONDUCT]: 'Will the exporter export using their code of conduct?',
     [YEARS_EXPORTING]: 'How long the business has been exporting for',
+    [IS_PARTY_TO_CONSORTIUM]: 'Party to any consortium in connection with the export contract(s)?',
+    [IS_MEMBER_OF_A_GROUP]: 'Member of a group which may have a part in negotiating the contract(s)?',
```

Comment:
Are these content strings different to the summary list? Just as these XLSX specific content strings are only used if they differ from the summary list (which most of them do now)

Reply:
these are different to the summary list one